package.json

```diff
@@ -10,15 +10,18 @@
   "scripts": {
     "preversion": "git pull && npm test",
     "version": "node bin/bump-dependencies.js && git add package.json",
-    "postversion": "git push && git push --tags",
+    "postversion": "git push && git push --tags && npm run deploy-boilerplate",
```

Comment:
We're using Travis to help us release `cli/core/schema` packages to npm. Maybe we want to do that too with the boilerplate? IE: Whenever there's a new tag on GitHub, Travis builds a new boilerplate and upload it to S3.

Reply:
The process is pretty much the same. We still need to run `npm version` locally. It just that `npm version` now does less -- it bumps version, adds a git tag, pushes to GitHub, and that's it. The rest of the deployment is done by Travis. So my point is, I think it's better to keep `npm version` doing as little as possible.

Travis supports [S3 deployment](https://docs.travis-ci.com/user/deployment/s3/). It's likely we can add another `provider` in `.travis.yml` to do the boilerplate deployment. This is not a blocker -- we can definitely leave it to another PR.

Reply:
Look to the playbook for an example where we do publish to S3 from Travis. :-)

Reply:
See https://github.com/zapier/playbook/blob/master/.travis.yml#L8